fplus-http-server/src/router/application.rs

```diff
@@ -205,7 +220,10 @@ pub async fn approve_storage_providers(
         )
         .await
     {
-        Ok(_) => HttpResponse::Ok().body(serde_json::to_string_pretty("Success").unwrap()),
+        Ok(_) => match serde_json::to_string_pretty("Success") {
```

Comment:
to_string_pretty won't fail if passed a static string. It's a good candidate for .expect.